main.go

```diff
@@ -57,21 +77,30 @@ func main() {
 	fmt.Printf("Found %d vulnerabilities \n", len(vs))
 	highSevNumber := len(store["High"]) + len(store["Critical"]) + len(store["Defcon1"])
 
-	iteratePriorities(func(sev string) {
+	iteratePriorities(clairOutput, func(sev string) {
 		for _, v := range store[sev] {
 			fmt.Printf("%s: [%s] \n%s\n%s\n", v.Name, v.Severity, v.Description, v.Link)
 			fmt.Println("-----------------------------------------")
 		}
 	})
-	iteratePriorities(func(sev string) { fmt.Printf("%s: %d\n", sev, len(store[sev])) })
+	iteratePriorities(priorities[0], func(sev string) { fmt.Printf("%s: %d\n", sev, len(store[sev])) })
 
 	if highSevNumber > threshold {
 		os.Exit(1)
 	}
 }
 
-func iteratePriorities(f func(sev string)) {
+func iteratePriorities(output string, f func(sev string)) {
+	filtered := true
```

Comment:
What if we just iterate over a subslice of `priorities` starting from the index of `output`? It may improve readability.